obj/HolyShift.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
You still need to untrack those files since they where pushed after .gitignore

(adding things to .gitignore does not work retroactively)